code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -176,18 +185,19 @@
 		update_appearance()
 		return
 
+/obj/machinery/atmospherics/components/unary/thermomachine/process_atmos_heating()
```

Comment:
What's the difference between the two ``process_atmos_heating()``?

Reply:
Still actively making changes, this will change very soon.